### PR TITLE
Doc rewrites for CDIOperatorDown runbook

### PR DESCRIPTION
**Pull request by davozeni**
Rewrote the CDIOperatorDown runbook for downstream conversion.

Signed-off-by: davozeni <davozeni@redhat.com>

### CODE REVIEW

File: docs/runbooks/CDIOperatorDown.md [hunk 1]
```diff
@@ -1,31 +1,46 @@
+<!-- Edited by davozeni, 10 Nov 2022 -->
+
 # CDIOperatorDown
 
 ## Meaning
 
-The CDI Operator is responsible for deploying and managing the CDI infrastructure components, such as the DataVolume/PVC controllers which help users build Virtual Machine Disks on PVCs.  
-This alert fires when the CDI operator is down.
+This alert fires when the containerized data importer (CDI) Operator is down.
```
Comment (apinnick) [inline docs/runbooks/CDIOperatorDown.md:7]: "containerized data importer" should be capitalized in this context.

File: docs/runbooks/CDIOperatorDown.md [hunk 2]
```diff
@@ -1,31 +1,46 @@
...
+The CDI Operator deploys and manages the CDI infrastructure components, such as data volume and persistent volume claim (PVC) controllers. These controllers help users build virtual machine disks on PVCs.
 
 ## Impact
 
-With CDI Operator down, the dependant infrastructure components may not deploy at all or fail to stay in a required, opinionated state.  
-As a result, the CDI installation is not 100% operational in the cluster.
+With the CDI Operator down, the dependant infrastructure components might fail to deploy or fail to stay in the required state.
```
Comment (apinnick) [inline docs/runbooks/CDIOperatorDown.md:12]: I think we can make this more concise:

"The CDI components might fail to deploy or to stay in a required state. The CDI installation might not function correctly."

**Review by apinnick — CHANGES_REQUESTED** [review]
Just a couple small fixes

File: docs/runbooks/CDIOperatorDown.md (hunk 2 above)
Comment (davozeni) [reply docs/runbooks/CDIOperatorDown.md]: Thank you for the suggestion. This is definitely more readable. Changed it in a second commit.

**Pull request merged**